test/data/REF_L_188298_configuration.xml

```diff
@@ -0,0 +1,43 @@
+<Reduction>
+ <instrument_name>REFL</instrument_name>
+<generator>RefRed</generator>
+ <DataSeries>
+  <RefLData>
+   <peak_selection_type>narrow</peak_selection_type>
+   <from_peak_pixels>129</from_peak_pixels>
+   <to_peak_pixels>141</to_peak_pixels>
+   <background_flag>True</background_flag>
+   <back_roi1_from>126</back_roi1_from>
+   <back_roi1_to>144</back_roi1_to>
+   <back_roi2_from>0</back_roi2_from>
+   <back_roi2_to>0</back_roi2_to>
+   <clocking_from>5</clocking_from>
+   <clocking_to>295</clocking_to>
+   <tof_range_flag>True</tof_range_flag>
+   <from_tof_range>41761.0114019</from_tof_range>
+   <to_tof_range>55052.5300504</to_tof_range>
+   <from_q_range>0.00933221394668</from_q_range>
+   <to_q_range>0.0123003643133</to_q_range>
+   <from_lambda_range>10.69</from_lambda_range>
+   <to_lambda_range>14.09</to_lambda_range>
+   <incident_angle>1.19907505794</incident_angle>
+   <data_sets>188299</data_sets>
```

Comment:
this xml is loading `188299`, not `188298`

Reply:
I better rename the file